profiles/cdl.pjs

```diff
@@ -1,6 +1,7 @@
 {
     "name": "cdl", 
     "type": "cdl",
+    "provider": "cdl",
```

Comment:
I was wrong about needing this line. It will pull the provider abbreviation from the CouchDB ingestion doc `provider` property, not the provider profile doc.